docs/_includes/conditional-include-directive.adoc

```diff
@@ -0,0 +1,85 @@
+////
+== Conditional Include Directive
+
+- User manual
+////
+You can include or exclude lines of text in your document from being processing by specifying three conditional include directives:
+
+* ifdef
+* ifndef
+* ifeval
```

Comment:
:+1: 

Let's use the term "process content" instead of "include content" for this section since "include" has another meaning in AsciiDoc. Technically, all the content is still read, it just isn't processed (and hence thrown away).